### PR TITLE
New UI changes

### PR DESCRIPTION
**Pull request by karlbeckettbjss**

### CODE REVIEW

File: src/main/java/uk/gov/dvsa/mot/fixtures/mot/TesterDoesStepDefinitions.java [hunk 1]
```diff
@@ -456,8 +476,8 @@ private void startMotTest(String registration, String vin, String siteName, bool
             }
 
             if (colour.isPresent()) {
-                //And I click the "Change" link for the colour
-                driverWrapper.clickLink("th", "Colour", "../td/", "Change");
+                //And I click the "Change" link for the colour by clicking the link containing change-under-test/colour
```
Comment (ibrahimjassat) [inline src/main/java/uk/gov/dvsa/mot/fixtures/mot/TesterDoesStepDefinitions.java]: Does not need the extra bit at the end

File: src/main/java/uk/gov/dvsa/mot/fixtures/mot/TesterDoesStepDefinitions.java [hunk 2]
```diff
@@ -470,8 +490,8 @@ private void startMotTest(String registration, String vin, String siteName, bool
             }
 
             if (fuelType.isPresent() || capacity.isPresent()) {
-                //And I click the "Change" link for the engine
-                driverWrapper.clickLink("th", "Engine", "../td/", "Change");
+                //And I click the "Change" link for the engine by clicking the link containing change-under-test/engine
```
Comment (ibrahimjassat) [inline src/main/java/uk/gov/dvsa/mot/fixtures/mot/TesterDoesStepDefinitions.java]: Does not need the extra bit at the end

File: src/main/resources/features/theme 1/05a - tester does.feature [hunk 3]
```diff
@@ -16,15 +16,16 @@ Feature: 05a - Tester does...
...
-    And I check the vehicle summary section of the test summary has "VIN/Chassis number" of {vin1}
-    And I check the brake results section of the test summary is "Pass"
-    And I check the dangerous failures section of the test summary has "None recorded"
-    And I check the major failures section of the test summary has "None recorded"
-    And I check the minors section of the test summary has "None recorded"
-    And I check the prs section of the test summary has "None recorded"
-    And I check the advisory section of the test summary has "None recorded"
+    And I check the vehicle summary section of the test summary has "Result" of "PASS"
+    And I check the brake test summary section has "Brake results overall" of "Pass"
+    And I check the registration plate {registration1} is shown within the registration number span text
```
Comment (ibrahimjassat) [inline src/main/resources/features/theme 1/05a - tester does.feature:19]: Ordering need to change, should do the checks from top to bottom

File: src/main/resources/features/theme 1/05a - tester does.feature [hunk 4]
```diff
@@ -49,15 +50,18 @@ Feature: 05a - Tester does...
...
     Then The page title contains "MOT test summary"
-    And I check the test information section of the test summary is "Fail"
-    And I check the vehicle summary section of the test summary has "Registration number" of {registration1}
-    And I check the vehicle summary section of the test summary has "VIN/Chassis number" of {vin1}
-    And I check the brake results section of the test summary is "Fail"
-    And I check the dangerous failures section of the test summary has "None recorded"
+
+
+    And I check the vehicle summary section of the test summary has "Result" of "FAIL"
+    And I check the registration plate {registration1} is shown within the registration number span text
```
Comment (ibrahimjassat) [inline src/main/resources/features/theme 1/05a - tester does.feature:53]: See above

File: src/main/resources/features/theme 1/05a - tester does.feature [hunk 5]
```diff
@@ -86,25 +90,27 @@ Feature: 05a - Tester does...
     And I press the "Review test" button
 
     Then The page title contains "MOT test summary"
-    And I check the test information section of the test summary is "Fail"
-    And I check the vehicle summary section of the test summary has "Registration number" of {registration1}
-    And I check the vehicle summary section of the test summary has "VIN/Chassis number" of {vin1}
-    And I check the brake results section of the test summary is "Pass"
+    And I check the vehicle summary section of the test summary has "Result" of "FAIL"
```
Comment (ibrahimjassat) [inline src/main/resources/features/theme 1/05a - tester does.feature]: See above comment

File: src/main/resources/features/theme 1/05a - tester does.feature [hunk 6]
```diff
@@ -134,15 +140,15 @@ Feature: 05a - Tester does...
...
-    And I check the test information section of the test summary is "Fail"
-    And I check the vehicle summary section of the test summary has "Registration number" of {registration1}
-    And I check the vehicle summary section of the test summary has "VIN/Chassis number" of {vin1}
-    And I check the brake results section of the test summary is "Not tested"
-    And I check the major failures section of the test summary has "Brake performance not tested"
-    And I check the dangerous failures section of the test summary has "None recorded"
-    And I check the minors section of the test summary has "None recorded"
-    And I check the prs section of the test summary has "None recorded"
-    And I check the advisory section of the test summary has "None recorded"
+    And I check the vehicle summary section of the test summary has "Result" of "FAIL"
```
Comment (ibrahimjassat) [inline src/main/resources/features/theme 1/05a - tester does.feature]: See above comment

File: src/main/resources/features/theme 1/05a - tester does.feature [hunk 7]
```diff
@@ -171,10 +177,10 @@ Feature: 05a - Tester does...
     And I press the "Review test" button
 
     Then The page title contains "MOT test summary"
-    And I check the test information section of the test summary is "Pass"
-    And I check the vehicle summary section of the test summary has "Registration number" of {registration1}
-    And I check the vehicle summary section of the test summary has "VIN/Chassis number" of {vin1}
-    And I check the brake results section of the test summary is "Pass"
+    And I check the vehicle summary section of the test summary has "Result" of "PASS"
+    And I check the registration plate {registration1} is shown within the registration number span text
```
Comment (ibrahimjassat) [inline src/main/resources/features/theme 1/05a - tester does.feature:178]: See above comment

File: src/main/resources/features/theme 1/05c - tester does.feature [hunk 8]
```diff
@@ -116,10 +117,14 @@ Feature: 05c - Tester does...
...
     Then The page title contains "MOT test summary"
-    And I check the test information section of the test summary is "Fail"
-    And I check the vehicle summary section of the test summary has "Registration number" of {registration1}
-    And I check the vehicle summary section of the test summary has "VIN/Chassis number" of {vin1}
-    And I check the vehicle summary section of the test summary has "Colour" of "Blue"
+    And I check the vehicle summary section of the test summary has "Result" of "FAIL"
+    And I check the registration plate {registration1} is shown within the registration number span text
+    And I check the VIN {vin1} is shown within the VIN span text
+
+
```
Comment (ibrahimjassat) [inline src/main/resources/features/theme 1/05c - tester does.feature:122]: Remove extra line

File: src/main/resources/features/theme 1/07 - tester records contingency.feature [hunk 9]
```diff
@@ -29,10 +29,10 @@ Feature: 07 - Tester records Contingency Test
     And I press the "Review test" button
 
     Then The page title contains "MOT test summary"
-    And I check the test information section of the test summary is "Pass"
+    And I check the vehicle summary section of the test summary has "Result" of "PASS"
```
Comment (ibrahimjassat) [inline src/main/resources/features/theme 1/07 - tester records contingency.feature]: Wording could be improving

File: src/main/resources/features/theme 1/07 - tester records contingency.feature [hunk 10]
```diff
@@ -71,10 +71,10 @@ Feature: 07 - Tester records Contingency Test
...
```
Comment (ibrahimjassat) [inline src/main/resources/features/theme 1/07 - tester records contingency.feature]: Wording could be improved

File: src/main/resources/features/theme 1/12a - vehicle examiner.feature [hunk 11]
```diff
@@ -279,17 +279,17 @@ Feature: 12a - Vehicle Examiner
     And I check the case outcome "Disciplinary action report" is saved
 
 
-  Scenario: VE re-inspection Fail to Satutory Appeal Verified Pass
+  Scenario: VE re-inspection Fail to Statutory Appeal Verified Pass
```
Comment (ibrahimjassat) [inline src/main/resources/features/theme 1/12a - vehicle examiner.feature:282]: Well done :D

**Comment by karlbeckettbjss** [comment]
Green run's complete

File: src/main/resources/features/theme 1/07 - tester records contingency.feature (hunk 9 above)
Comment (karlbeckettbjss) [reply src/main/resources/features/theme 1/07 - tester records contingency.feature]: I didn't get time to change this

File: src/main/resources/features/theme 1/07 - tester records contingency.feature (hunk 10 above)
Comment (karlbeckettbjss) [reply src/main/resources/features/theme 1/07 - tester records contingency.feature]: I didn't get time to change this

File: src/main/resources/features/theme 1/05a - tester does.feature [hunk 12]
```diff
@@ -49,15 +50,17 @@ Feature: 05a - Tester does...
...
-    And I check the test information section of the test summary is "Fail"
-    And I check the vehicle summary section of the test summary has "Registration number" of {registration1}
-    And I check the vehicle summary section of the test summary has "VIN/Chassis number" of {vin1}
-    And I check the brake results section of the test summary is "Fail"
-    And I check the dangerous failures section of the test summary has "None recorded"
+
+    And I check the registration plate {registration1} is shown within the registration number span text
+    And I check the VIN {vin1} is shown within the VIN span text
+    And I check the vehicle summary section of the test summary has "Result" of "FAIL"
+
```
Comment (ibrahimjassat) [inline src/main/resources/features/theme 1/05a - tester does.feature]: Remove this line

File: src/main/resources/features/theme 1/05a - tester does.feature [hunk 13]
```diff
@@ -49,15 +50,17 @@ Feature: 05a - Tester does...
...
+    And I check the defect section has "Dangerous failures" with value "None recorded"
     And I check the major failures section of the test summary has "Parking brake efficiency below requirements"
-    And I check the minors section of the test summary has "None recorded"
-    And I check the prs section of the test summary has "None recorded"
-    And I check the advisory section of the test summary has "None recorded"
+    And I check the defect section has "Minors" with value "None recorded"
+    And I check the defect section has "PRS" with value "None recorded"
+    And I check the defect section has "Advisory text" with value "None recorded"
     And I press the "Save test result" button
     And The page title contains "MOT test complete"
```
Comment (ibrahimjassat) [inline src/main/resources/features/theme 1/05a - tester does.feature:64]: Would have the new line here as this goes onto a new page

File: src/main/resources/features/theme 1/05b - tester does.feature [hunk 14]
```diff
@@ -19,10 +19,12 @@ Feature: 05b - Tester does...
...
-    And I check the test information section of the test summary is "Pass"
-    And I check the vehicle summary section of the test summary has "Registration number" of {registration1}
-    And I check the vehicle summary section of the test summary has "VIN/Chassis number" of {vin1}
-    And I check the brake results section of the test summary is "Pass"
+    And I check the registration plate {registration1} is shown within the registration number span text
+    And I check the VIN {vin1} is shown within the VIN span text
+    And I check the vehicle summary section of the test summary has "Result" of "PASS"
+    And I check the brake test summary section has "Brake results overall" of "Pass"
+
+
```
Comment (ibrahimjassat) [inline src/main/resources/features/theme 1/05b - tester does.feature]: Remove empty line

**Review by ibrahimjassat — CHANGES_REQUESTED** [review]

**Review by ibrahimjassat — APPROVED** [review]

**Pull request merged**